Fix flake8 style guide failures

We are using the lowercase letter `L` as a variable name with can be confused with `I` or `1`.

```
tests/test_membership.py:34:50: E741 ambiguous variable name 'l'
tests/test_membership.py:35:50: E741 ambiguous variable name 'l'
tests/test_membership.py:80:9: E741 ambiguous variable name 'l'
tests/test_membership.py:103:9: E741 ambiguous variable name 'l'
tests/test_membership.py:120:9: E741 ambiguous variable name 'l'
tests/test_membership.py:139:9: E741 ambiguous variable name 'l'
tests/test_membership.py:161:9: E741 ambiguous variable name 'l'
tests/test_batch_change.py:47:13: E741 ambiguous variable name 'l'
tests/test_batch_change.py:66:9: E741 ambiguous variable name 'l'
tests/test_batch_change.py:246:9: E741 ambiguous variable name 'l'
tests/test_records.py:32:50: E741 ambiguous variable name 'l'
tests/test_records.py:113:9: E741 ambiguous variable name 'l'
tests/test_records.py:139:9: E741 ambiguous variable name 'l'
tests/test_records.py:179:9: E741 ambiguous variable name 'l'
tests/test_zones.py:48:46: E741 ambiguous variable name 'l'
tests/test_zones.py:110:9: E741 ambiguous variable name 'l'
tests/test_zones.py:144:9: E741 ambiguous variable name 'l'
```

--- a/tox.ini
+++ b/tox.ini
@@ -35,7 +35,8 @@ commands =
 [testenv:check]
 deps =
     docutils
-    flake8
+    flake8==3.7.9
+
 skip_install = true
 commands =
     flake8 src tests setup.py

